src/NUnitFramework/framework/Internal/TestExecutionContext.cs

```diff
@@ -399,9 +399,7 @@ public CultureInfo CurrentCulture
             set
             {
                 _currentCulture = value;
-#if NETCORE
-                CultureInfo.CurrentCulture = _currentCulture;
-#elif !NETCF
+#if !NETCF && !PORTABLE
```

Comment:
We cannot set the CurrentCulture in this profile. If anyone knows another way...